guides/operator/orca-redis-to-sql.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: single
+title:  "Orca: Redis to SQL"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Orca's execution state is stored in Redis by default, but can be configured for SQL. In this topology, Redis will still be required for the work queue.
+
+This guide will go over MySQL setup, how to configure Orca, as well as how to perform a zero-downtime migration from Redis to SQL.
+
+## MySQL Setup
+
+Orca ships with MySQL drivers by default, but you can include your own JDBC drivers on the classpath if you need to connect to a different database.
+
+First, you'll need to create the schema:
+
+```sql
+CREATE SCHEMA `orca` DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ;
+```
+
+Orca is configured to support a `service` user and a `migration` user:
+
+```sql
+GRANT SELECT, INSERT, UPDATE, DELETE, EXECUTE, SHOW VIEW ON `orca`.* TO 'orca_service'@'%'
+
+GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, REFERENCES, INDEX, ALTER, LOCK TABLES, EXECUTE, SHOW VIEW ON `orca`.* TO 'orca_migrate'@'%'
+```
+
+When Orca starts up, it will perform database migrations to ensure its running the correct schema.
+
+### Amazon Aurora
+
+At Netflix, we use Amazon Aurora MySQL and have configured it with the following settings, missing values are defaults.
+Your mileage may vary.
+
+#### Aurora Parameter Groups
+
+- *binlog_cache_size*: `32768`
+- *default_tmp_storage_engine*: `InnoDB`
+- *general_log*: `0`
+- *innodb_adaptive_hash_index*: `0`
+- *innodb_buffer_pool_size*: `{DBInstanceClassMemory*3/4}`
+- *key_buffer_size*: `16777216`
+- *log_queries_not_using_indexes*: `0`
+- *log_throttle_queries_not_using_indexes*: `60`
+- *long_query_time*: `0.5`
+- *max_allowed_packet*: `25165824`
+- *max_binlog_size*: `134217728`
+- *query_cache_size*: `{DBInstanceClassMemory/24}`
+- *query_cache_type*: `1`
+- *read_buffer_size*: `262144`
+- *slow_query_log*: `1`
+- *sync_binlog*: `1`
+- *tx_isolation*: `READ-COMMITTED`
```

Comment:
I think our use of `READ-COMMITTED` is worth noting outside of this block as it is both non-default and essential. While orca attempts to set this at the session level regardless, it may be of interest to DBE's operating mysql for orca.